commit 2d6da65598b90480b4a62c4633eda035ea51681f
Author: David King <amigadave@amigadave.com>
Date:   Wed Jun 27 06:36:00 2018 +0100

    python: support Python 3.7 and 3.8
    
    Add 3.7 and 3.8 to the list of accepted Python 3 versions.

diff --git a/cmake/FindPython3Interp.cmake b/cmake/FindPython3Interp.cmake
index c6cbe3d..8e5e409 100644
--- a/cmake/FindPython3Interp.cmake
+++ b/cmake/FindPython3Interp.cmake
@@ -39,7 +39,7 @@
 
 unset(_Python3_NAMES)
 
-set(_Python3_VERSIONS 3.6 3.5 3.4 3.3 3.2 3.1 3.0)
+set(_Python3_VERSIONS 3.8 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0)
 
 if(Python3Interp_FIND_VERSION)
     if(Python3Interp_FIND_VERSION_COUNT GREATER 1)
